smswebapp/settings/developer.py

```diff
@@ -53,3 +53,10 @@
     warnings.warn(
         'JWPLATFORM_EMBED_PLAYER_KEY is not set. Embedded players will not function.',
         RuntimeWarning)
+
+SMS_OAUTH2_CLIENT_ID = 'smswebapp'
+SMS_OAUTH2_CLIENT_SECRET = 'smssecret'
+SMS_OAUTH2_TOKEN_URL = 'http://hydra:4444/oauth2/token'
+LOOKUP_ROOT = 'http://lookupproxy:8080/'
+
+JWPLATFORM_EMBED_PLAYER_KEY = '3XZDkSrY'
```

Comment:
This is a deployment-specific configuration and I'd prefer to keep this repository deployment neutral if possible.

(Aside: I was also worried that this was a secret but the comment in settings_deployment.py in rjw57/sms-deploy@ca09effc suggests that past-Rich didn't think it was. It also suggests that past-Rich thought this setting should be a deployment-specific one. Past-Rich knew this code far better than present-Rich so I'm inclined to trust his judgement.)